stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -694,26 +694,28 @@ extension Unicode.Scalar.Properties {
   internal func _applyMapping(_ u_strTo: _U_StrToX) -> String {
     // TODO(String performance): Stack buffer first and then detect real count
     let count = 64
-    var array = Array<UInt16>(repeating: 0, count: count)
-    let len: Int = array.withUnsafeMutableBufferPointer { bufPtr in
-      return _scalar.withUTF16CodeUnits { utf16 in
-        var err = __swift_stdlib_U_ZERO_ERROR
+    var length: Int = 0
```

Comment:
You don't need to compute this outside of the array initializer — the array will end up having the correct count because of the way you set the initialized count at line 715.

Reply:
Makes sense. Fixed.